packages/jellyfish-api-core/src/category/account.ts

```diff
@@ -235,6 +235,22 @@ export class Account {
   async utxosToAccount (payload: UtxosToAccountPayload, utxos: UtxosToAccountUTXO[] = []): Promise<string> {
     return await this.client.call('utxostoaccount', [payload, utxos], 'number')
   }
+
+  /**
+   * Creates and submits to a local node and transfer transaction from the specified account to the specfied accounts.
+   * Optionally, specific UTXOs to spend to create that transaction.
+   *
+   * @param {string} from
+   * @param {AccountToAccountPayload} payload
+   * @param {string} payload[address]
+   * @param {AccountToAccountUTXO[]} [utxos=[]]
+   * @param {string} [utxos.txid]
+   * @param {number} [utxos.vout]
+   * @return {Promise<string>}
+   */
+  async accountToAccount (from: string, payload: AccountToAccountPayload, utxos: AccountToAccountUTXO[] = []): Promise<string> {
```

Comment:
The third parameter needs to be changed to option right?

Reply:
I think for this situation options would be cleaner. Since `utxos` is an option that is `option`al.